methods_results.Rmd

```diff
@@ -10,6 +10,8 @@ csl: Rmd/elsevier-harvard_rjt.csl
 
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE)
+library("drake")
```

Comment:
If you are making the pdf through the plan, you don't need to add any packages here, unless they are not already loaded by the plan

Reply:
ok, forgot to delete it again. Do I need to loadd(figure) thingi or is readd(figure) enough

Reply:
For anything you only use once (probably all figures, most tables) readd is perfect.
If you want to use something repeatedly, usually this would be in in-line code, use loadd

Reply:
Ok, that makes sense.